### PR TITLE
APS-REVAMP-01 DOCS 

### PR DESCRIPTION
**Pull request by mahendra-aps**
Docs generation adjustments as discussed

### CODE REVIEW

File: core/src/Rpc/Client.cs [hunk 1]
```diff
@@ -133,15 +133,40 @@ public Task<ProtoMember> UpdateMember(IList<MemberOperation> operations)
...
                     {
-                        RecoveryRule = new RecoveryRule { PrimaryAgent = response.MemberId }
+                        KeyId = signer.GetKeyId(),
+                        MemberId = MemberId,
+                        Signature_ = signer.Sign(update)
                     }
                 })
-                .FlatMap(opration => UpdateMember(new List<MemberOperation> { opration }));
+                .ToTask())
+                );
```
[the remark this answers is not in the document]
Comment (mahendra-aps) [reply core/src/Rpc/Client.cs]: done

File: core/src/Rpc/Client.cs [hunk 2]
```diff
@@ -133,15 +133,40 @@ public Task<ProtoMember> UpdateMember(IList<MemberOperation> operations)
         /// <returns>a task</returns>
         public Task UseDefaultRecoveryRule()
         {
-            return gateway(authenticationContext()).GetDefaultAgentAsync(new GetDefaultAgentRequest())
-                .ToTask(response => new MemberOperation
+
```
[the remark this answers is not in the document]
Comment (mahendra-aps) [reply core/src/Rpc/Client.cs]: done

File: core/src/TokenRequests/TokenRequest.cs [hunk 3]
```diff
@@ -58,13 +59,13 @@ public static AccessBuilder AccessTokenRequestBuilder(AccountResourceList list)
...
         }
-        
+
         /// <summary>
         /// Create a Builder instance for a funds confirmation request.
         /// </summary>
         /// <param name="bankId">bank ID</param>
         /// <param name="account">the user's account</param>
-        /// <param name="data">optional customer data</param>
+        /// <param name="data">customer data</param>
```
[the remark this answers is not in the document]
Comment (mahendra-aps) [reply core/src/TokenRequests/TokenRequest.cs]: done

File: tpp/src/Member.cs [hunk 4]
```diff
@@ -371,6 +373,7 @@ public Task<Transfer> RedeemToken(Token token, TransferDestination destination)
         /// <param name="refId">the reference id of the transfer</param>
         /// <returns>a transfer record</returns>
         /// <remarks>amount, currency, description, destination and refId are nullable</remarks>>
+        [Obsolete("Depricated")]
```
[the remark this answers is not in the document]
Comment (mahendra-aps) [reply tpp/src/Member.cs]: done

File: core/src/TokenRequests/TokenRequest.cs [hunk 5]
```diff
@@ -97,6 +98,52 @@ public static TransferBuilder TransferTokenRequestBuilder(double amount, string
...
+        /// 
+        /// </summary>
+        /// <param name="transfers">transfers list of transfers</param>
+        /// <param name="totalAmount">total amount irrespective of currency. Used for redundancy check.</param>
+        /// <param name="source">source account for all transfer</param>
+        /// <returns>Builder instance</returns>
+        public static BulkTransferBuilder BulkTransferRequestBuilder(
+                IList<BulkTransferBody.Types.Transfer> transfers,
+                double totalAmount,
+                TransferEndpoint source)
```
[the remark this answers is not in the document]
Comment (mahendra-aps) [reply core/src/TokenRequests/TokenRequest.cs]: done

File: core/src/TokenRequests/TokenRequest.cs [hunk 6]
```diff
@@ -490,13 +551,51 @@ public TransferBuilder SetConfirmFunds(bool confirmFunds)
             }
         }
 
+        public class BulkTransferBuilder : Builder<BulkTransferBuilder>
+        {
+            internal BulkTransferBuilder(
+                    IList<BulkTransferBody.Types.Transfer> transfers,
+                    double totalAmount,
+                    TransferEndpoint source)
```
[the remark this answers is not in the document]
Comment (mahendra-aps) [reply core/src/TokenRequests/TokenRequest.cs]: done

File: tpp/src/Member.cs [hunk 7]
```diff
@@ -328,7 +330,7 @@ public Task<Transfer> RedeemToken(Token token, TransferDestination destination)
             {
                 payload.RefId = refId;
             }
-            else if (!string.IsNullOrEmpty(token.Payload.RefId))
+            else if (!string.IsNullOrEmpty(token.Payload.RefId) && amount == null)
```
[the remark this answers is not in the document]
Comment (gianluca-token) [reply tpp/src/Member.cs]: You only specify amount if you want to redeem for less than the full amount

**Pull request closed without merging**